cmd/fs/helpers.go

```diff
@@ -2,55 +2,32 @@ package fs
 
 import (
 	"context"
-	"fmt"
-	"runtime"
 	"strings"
 
 	"github.com/databricks/cli/cmd/root"
 	"github.com/databricks/cli/libs/filer"
 )
 
-type Scheme string
-
-const (
-	DbfsScheme  = Scheme("dbfs")
-	LocalScheme = Scheme("file")
-	NoScheme    = Scheme("")
-)
-
 func filerForPath(ctx context.Context, fullPath string) (filer.Filer, string, error) {
-	parts := strings.SplitN(fullPath, ":/", 2)
-	if len(parts) < 2 {
-		return nil, "", fmt.Errorf(`no scheme specified for path %s. Please specify scheme "dbfs" or "file". Example: file:/foo/bar or file:/c:/foo/bar`, fullPath)
+	// Split path at : to detect any file schemes
+	parts := strings.SplitN(fullPath, ":", 2)
+
+	// If dbfs file scheme is not specified, then it's a local path
+	if len(parts) < 2 || parts[0] != "dbfs" {
+		f, err := filer.NewLocalClient("")
+		return f, fullPath, err
 	}
```

Comment:
This is not correct though; if folks specify foobar:/bla` it shouldn't be interpreted as a local path.

If there is a `:` in the path they can always use `./foo:bar` to specify it and it should work.

As long as the `parts[0]` is all `\w` characters, it should be "dbfs". If it isn't, we should return an error.

Otherwise typos on the scheme can cause issues.